### PR TITLE
P5 soccer workshop

### PR DESCRIPTION
**Pull request by jonleung**
@paked, can you do the same thing as you did for the previous workshop and submit a pull request for any issues that you see in this workshop?

It was **super** helpful last time.

Thanks so much!
Jonathan

### CODE REVIEW

**Comment by jonleung** [comment]
Fixes #229

**Comment by jonleung** [comment]
@paked, gah I forgot to fix the failing marcop checks! Going to do that shortly!

**Comment by paked** [comment]
@jonl I'll review these changes ASAP :) :heart:

**Comment by paked** [comment]
@jonleung Do you want me to fix these merge issues?

**Comment by jonleung** [comment]
@paked, I just fixed the markdown issues! I can rebase after your pull request!

**Comment by jonleung** [comment]
@MaxWofford, hey, can you help review this pull request for me?

**Comment by MaxWofford** [comment]
@jonleung Sure!

**Comment by jonleung** [comment]
Thanks @MaxWofford!

**Comment by jonleung** [comment]
@MaxWofford, do you have the bandwidth to actually submit a pull request for any changes that you think should be made instead of making comments? That would be really appreciated. Thanks!

File: playbook/workshops/soccer/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+1. The code for soccer example at the top of the page
+2. The Dodge Workshop
+
+Think of this section like the like the topping bar of an ice cream store. You
+can sprinkle as many or as little of the below features on the maze game that
+you built in [Part 1](#part-1-single-player-soccer) in any order you please!
+
+```
+TODO: Adding Functionality
```
Comment (MaxWofford) [inline playbook/workshops/soccer/README.md]: @jonleung Should this be here?

**Comment by MaxWofford** [comment]
@jonleung Will merge once tests run. Here's the changes: a93e684f95060e0b7f1549a4abccd86fa91f83e2

**Comment by jonleung** [comment]
@MaxWofford thaaaaaanks! I will try to make sure to give you more advance notice in the future!

**Pull request merged**